### PR TITLE
Compare dev_texadactyl to master !

### PR DESCRIPTION
**Pull request by texadactyl**
Hopefully, the conflicts will be much fewer.

### CODE REVIEW

File: hyperseti/normalize.py [hunk 1]
```diff
@@ -66,18 +66,18 @@ def normalize(data,  mask=None, poly_fit=0):
...
     t1p = time.time()
-    logger.info(f"Poly fit time: {(t1p-t0p)*1e3:2.2f}ms")
+    ### logger.info(f"Poly fit time: {(t1p-t0p)*1e3:2.2f}ms")
 
     flag_fraction =  N_flagged / N_tot
     flag_correction =  N_tot / (N_tot - N_flagged) 
-    logger.info(f"Flagged fraction: {flag_fraction:2.4f}")
-    if flag_fraction > 0.2:
-        logger.warning(f"High flagged fraction: {flag_fraction:2.3f}")
+    ### logger.info(f"Flagged fraction: {flag_fraction:2.4f}")
```
Comment (telegraphic) [inline hyperseti/normalize.py:73]: IIRC these were commented out due to a runtime issue with older environment -- can these be added back in? The high flagged fraction warning is important

File: setup.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-__version__ = "0.0.1"
+__version__ = "0.0.5"
 
 with open("hyperseti/version.py", "w") as fh:
```
Comment (telegraphic) [inline setup.py:5]: This is nice -- are there any circumstances where it could find the wrong file path?

File: test/test_with_voyager.py [hunk 3]
```diff
@@ -21,13 +21,13 @@
 from hyperseti import find_et
 from hyperseti.log import update_levels
 
-from file_defs import synthetic_fil, test_fig_dir, voyager_h5
+from .file_defs import synthetic_fil, test_fig_dir, voyager_h5
```
Comment (telegraphic) [inline test/test_with_voyager.py:24]: This relative import error is quite frustrating when testing (at least with my approach of running from the test directory on occasion).

Wondering whether test data should be moved elsewhere? In PyGDSM I opted to use astropy's download utility, which downloads to `~/.astropy/cache` by default:
https://github.com/telegraphic/pygdsm/blob/52eff5cd875d0938211201a8b3625b5b3836ec5d/pygdsm/component_data.py#L1

We could change download path to `~/.hyperseti`, or use a relative path to test directory. 

The current location of `/test` is the most logical for command-line development, but I think `pip install` may actually treat 'test' as a package and install it into `site-packages`. So could think about moving too

**Review by telegraphic — APPROVED** [review]
All changes look good to me. `tsdat` will be very useful for verifying against turbo_seti. 

I should take a look at tidying up the tests, but that's a separate issue to this PR.

File: hyperseti/normalize.py (hunk 1 above)
Comment (texadactyl) [reply hyperseti/normalize.py]: Agreed.

File: test/test_with_voyager.py (hunk 3 above)
Comment (texadactyl) [reply test/test_with_voyager.py:24]: I have always detested the inconsistency of "from .FILE" syntax in Python.  It works for pytest but it fails if you execute the file directly with python3.  In turbo_seti newer tests, I put them under the termporary directory, like this:
```
from tempfile import gettempdir
TESTDATA = gettempdir() + "/hyperseti_test_data/"
```
We could do that with test_data and test_figs in general.

**Pull request merged**